governance.qmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "R-Multiverse Governance"
+---
+
+The R-Multiverse project is run in a collaborative and open way.
+Portions of this governance policy have been inspired by that of the Apache Software Foundation.
+
+# The R-Multiverse Project
+
+The R-Multiverse project consists of the code and repositories that reside in the `r-multiverse` GitHub organization and the r-multiverse.org website.
+All repositories are open to everyone, with the exception of a private repository where security and conduct-related matters are shared between the administrators only.
+
+## R-Multiverse Volunteers
+
+We recognize four groups of individuals related to the project.
+
+### User
+
+A user is someone who uses R-Multiverse by downloading and installing packages from the R-Multiverse repositories.
+
+### Contributor
+
+A contributor is someone who seeks to make a modification to the R-Multiverse repositories and/or the GitHub organization.
+This would most commonly be by contributing a package at `r-multiverse/contributions`, where the contributor is often the maintainer of the contributed package.
+Contributions may also take the form of feedback in the form of bug reports and feature suggestions in the `r-multiverse/help` repository, or more generally by answering questions from the community in this repository.
+
+### Moderator
+
+A moderator is someone who has been invited by the R-Multiverse Management Committee (RMC) to take on a moderator role.
+This entails certains rights and associated responsibilities which are outlined in full in the Review Policy, which may evolve over time.
+A moderator may vet contributions and decide to admit or remove contributed packages from the repositories in accordance with the Review Policy.
+
+### Administrator
+
+An administrator is a member of the R-Multiverse Management Committee (RMC).
+Administrators have write access to all code repositories under the `r-multiverse` GitHub organization and maintain the infrastructure for the project.
+They have the right to cast votes on all proposals on community-related decisions, the right to appoint moderators, and the right to change the composition of the RMC itself.
+The RMC as a whole is the entity that controls the project, and is responsible for the continued shaping of this governance document and other policies, including the Review Policy and Code of Conduct.
+
+## R-Multiverse Management and Collaboration
+
+The R-Multiverse project is managed through a collaborative, consensus-based approach.
+
+### Communication
+
+Communication is always done publicly via GitHub.
+There are no private mailing lists and all decisions related to the project are made in public.
```

Comment:
Maybe mention that the private repo makes confidential conversations more traceable and able to be audited than a mailing list.